Library/Formula/nawk.rb

```diff
@@ -0,0 +1,22 @@
+class Nawk < Formula
+  desc "Brian Kernighan's pattern scanning and processing language"
+  homepage "https://github.com/junghans/nawk"
+  url "https://github.com/junghans/nawk/archive/20121220.tar.gz"
+  sha256 "50b7a138242f0d2f5b63b72ddfc48d94e60edc495fc4b6ee26a25051b08e5127"
+  head "https://github.com/junghans/nawk.git"
+
+  def install
+    make_args = ["YACC=bison -d -y"]
+    # have to run bison first or parallel build fails
+    system "make", *make_args, "ytab.o"
```

Comment:
You can't `*` other than the last argument on Ruby 1.8 (on Mountain Lion, which we support)